File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l = 0
        set_nums = set()
        for r in range(len(nums)):
            if nums[r] not in set_nums:
                set_nums.add(nums[r])
                nums[l] = nums[r]
                l+=1
        return len(set_nums)

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    findMaxConsecutiveOnes(nums) {
        let max =  0;
        let count = 0;
        for(let i=0;i < nums.length; i++){
            if(nums[i] === 1)
                count++;
            else
            {
                max = Math.max(count, max);
                count = 0;
            }
        }
        max = Math.max(count, max);
        return max;
    }
}
